### ISSUE TITLE
Misleading warning message if version check is skipped

### ISSUE THREAD

**Issue by lizrice** [aquasecurity/kube-bench#20]
If the executable(s) aren't on the path (pending issue #15) we skip the version check, but we get the following warning: 

`[WARN] kube-apiserver unsupported version`

This is misleading - we should distinguish between warning that the version is incorrect and warning that the version check has been skipped.

**Comment by ttousai** [comment]
Welcome back :)

**Comment by lizrice** [comment]
haha, thanks :-) 

Also, if the --verbose flag is on we get a lot of somewhat unnecessary error messages: 

```[ec2-user@k8s93-vm0 ~]$ ./kube-bench master --installation kubeadm --verbose
kube-apiserver: command not found in path, error: exec: "kube-apiserver": executable file not found in $PATH
kube-scheduler: command not found in path, error: exec: "kube-scheduler": executable file not found in $PATH
kube-controller-manager: command not found in path, error: exec: "kube-controller-manager": executable file not found in $PATH
kube-apiserver: command not found on path - version check skipped, error: exec: "kube-apiserver": executable file not found in $PATH
failed to run:[kube-apiserver --version], error: exec: "kube-apiserver": executable file not found in $PATH```

I don't think there's any need to look for all the binaries in the path, we only need to try the one we would use for the version check?

**Comment by ttousai** [comment]
I agree. The error handling is not as good as I would like it. I will work on this.

**Comment by lizrice** [comment]
Also now fixed I believe with #34
